packit_service/service/api/pull_from_upstream.py

```diff
@@ -0,0 +1,68 @@
+# Copyright Contributors to the Packit project.
+# SPDX-License-Identifier: MIT
+
+from http import HTTPStatus
+from logging import getLogger
+
+from flask_restx import Namespace, Resource
+
+from packit_service.models import (
+    SyncReleaseTargetModel,
+    SyncReleaseModel,
+    SyncReleaseJobType,
+)
+from packit_service.service.api.parsers import indices, pagination_arguments
+from packit_service.service.api.utils import (
+    response_maker,
+    get_sync_release_target_info,
+    get_sync_release_info,
+)
+
+logger = getLogger("packit_service")
+
+ns = Namespace("pull-from-upstream", description="Pull from upstream")
```

Comment:
Do you think we should differentiate between `pull-from-upstream` and  `propose-downstream` even if we use the same model for those?

But I agree that they will probably diverge even more in the future...

(= Alternatively, we can create a new `sync-release` endpoint, but I am really not sure here..;)

Reply:
> But I agree that they will probably diverge even more in the future...

Yes, that's the reason why I think it makes sense to have separate endpoints.